policy/modules/kernel/filesystem.te

```diff
@@ -165,6 +165,10 @@ type oprofilefs_t;
 fs_type(oprofilefs_t)
 genfscon oprofilefs / gen_context(system_u:object_r:oprofilefs_t,s0)
 
+type pidfs_t;
+fs_type(pidfs_t)
+genfscon pidfs_t / gen_context(system_u:object_r:pidfs_t,s0)
```

Comment:
`pidfs_t` looks wrong as filesystem-name, should probably be `pidfs`.

Also is there a difference compared to using `fs_use_task pidfs gen_context(system_u:object_r:pidfs_t,s0);`, which I use in my personal policy?